source/MaterialXCore/Value.cpp

```diff
@@ -232,6 +244,20 @@ template<class T> T Value::asA() const
     return typedVal->getData();
 }
 
+Value::ScopedFloatFormatting::ScopedFloatFormatting(FloatFormat format, int precision)
+    : _format(Value::getFloatFormat())
```

Comment:
For consistency, I'd place the colon and comma at line endings rather than line beginnings.